tools/rosbash/scripts/rosrun

```diff
@@ -66,15 +68,18 @@ if [[ ! $2 == */* ]]; then
     _perm="/111"
   fi
   debug "Searching for $2 with permissions $_perm"
-  exepathlist=(`find -L $catkin_package_libexec_dir $pkgdir -name $2 -type f  -perm $_perm ! -regex ".*$pkgdir\/build\/.*" | uniq`)
+  exepathlist="`find -L "${catkin_package_libexec_dirs[@]}" "$pkgdir" -name "$2" -type f  -perm "$_perm" ! -regex ".*$pkgdir\/build\/.*" | uniq`"
```

Comment:
Since `${catkin_package_libexec_dirs[@]}` might result in multiple paths `find -L "/path/foo /path/bar" ...` will fail.